glide.yaml

```diff
@@ -59,7 +59,7 @@ import:
 - package: github.com/asaskevich/govalidator/
   version: 5
 - package: github.com/DATA-DOG/godog
-  version: v0.6.2
+  version: v0.7.4
```

Comment:
Any specific reason we are going to the newer version? Or it is just to update the version of godog.

Reply:
@LalatenduMohanty Yes, in 0.7.4 there were beforeFeature() and afterFeature() hooks introduced, which I use to secure more isolation between individual features - until now if change in config for example was made in basic it went through all other features after the basic feature. Issue of mine for the hooks: https://github.com/DATA-DOG/godog/issues/97